extension/chrome/settings/setup.htm

```diff
@@ -120,8 +120,9 @@ <h1></h1>
       </div>
 
       <div class="line left">
-        <label data-test="input-step2bmanualcreate-save-passphrase">
-          <input type="checkbox" class="input_passphrase_save" value="" name="check" />
+        <label>
+          <input type="checkbox" class="input_passphrase_save" data-test="input-step2bmanualcreate-save-passphrase"
```

Comment:
I moved `data-test="input-step2bmanualcreate-save-passphrase"` from `<label>` to `<input>` to easily check for disabled status of the button. Or should we somehow disable the label itself? Does it make sense, @limonte ?

Reply:
I think disabling input is ok, and moving the data-test looks ok to me too.